pages/news.vue

```diff
@@ -13,15 +13,18 @@
     </div>
     <v-row>
       <v-col v-if="loaded">
+        <v-btn class="ma-2" medium rounded @click="tagSelection('reset')">
+          Reset filters
+        </v-btn>
```

Comment:
I'm wondering if instead of a Reset filters button, would the use of closable chips work well to deactivate a tag filter?
https://vuetifyjs.com/en/components/chips/#closable

Reply:
Thinking about this, I'm not sure how this would look, because if a tag was unselected (and closed), how would the user reselect it if they wanted to? My thought process was that all tags for each article are shown at all times for the user to see, and the selected tags are highlighted.